internal/ingress/controller/template/crossplane/crossplane_test.go

```diff
@@ -186,6 +186,8 @@ func TestCrossplaneTemplate(t *testing.T) {
 		tplConfig.Cfg.UpstreamKeepaliveTimeout = 200
 		tplConfig.Cfg.UpstreamKeepaliveRequests = 15
 
+		tplConfig.Cfg.CustomHTTPErrors = []int{400, 404, 501}
```

Comment:
On the upstream branch, https://github.com/kubernetes/ingress-nginx/blob/feature-go-crossplane/internal/ingress/controller/template/crossplane/crossplane_test.go#L164 exists.

You probably need to rebase and remove this line, and keep just your custom http errors (400, 404, etc)

Reply:
I didn't see that the test existed. I prefer to keep the original.